apps/schema/static/schema/swagger.yaml

```diff
@@ -76,6 +76,12 @@ paths:
   /api/v1/devices/status/:
     $ref: components/devices/deviceStatus.yaml
 
+  /api/v1/imports/shipments/:
+    $ref: components/shipmentImports/shipmentImports.yaml
+
+  /api/v1/imports/shipments/{shipment_import_id}/:
+    $ref: components/shipmentImports/shipmentImportsId.yaml
+
 components:
```

Comment:
Since you're adding a new tag, you should add it's name and description to the bottom of this swagger page